.github/workflows/tests.yml

```diff
@@ -21,6 +21,22 @@ jobs:
       run: cmake ..
     - name: build
       run: make
-    - name: run tests
-      run: ./tests
+    - name: run unit tests
+      run: ./tests    
+
+    # simulator runs
+    - name: run test-tiny
+      run: ./cgraphitti -c ../configfiles/test-tiny.xml
```

Comment:
I'm a little concerned about leaving the config files for regression testing in the configfiles directory; various people modify them for various purposes (this is not good practice, but...). Perhaps you could copy them into the `../Testing/RegressionTestComparison` directory?

Reply:
test config files have been copied to their own folder in the Testing directory